### ISSUE TITLE
Prepared statement "Codec not found for requested operation"

### ISSUE THREAD

**Issue by mmatloka** [websudos/phantom#347]
Hi,
I am testing phantom `0.14.2` and I have some problem with a prepared statement. I've got a statement where one comparision is a string and a second is an enum:
```
 select.p_where(_.id eqs ?).p_and(_.someEnum eqs ?)
```
result is:
```
Codec not found for requested operation: [varchar <-> scala.Enumeration$Val]
com.datastax.driver.core.exceptions.CodecNotFoundException: Codec not found for requested operation: [varchar <-> scala.Enumeration$Val]
```

**Comment by alexflav23** [comment]
Hi @mmatloka ,

This is well expected given the novelty of prepared statements, there is no codec inside the Java driver itself for Scala enums. Anyway, thank you for reporting this, we will fix it shortly.

Regards.

**Comment by alexflav23** [comment]
Hi @mmatloka,

This should now be fixed as of phantom `1.15.0` that was just published to Bintray alongside the new support for Reactive Streams.

Regards.

**Comment by mmatloka** [comment]
Hi @alexflav23 , are you sure its fixed? :) For me still occurs and I see in github history only commit related to Reactive Streams.

**Comment by alexflav23** [comment]
Hi @mmatloka,

You are correct, my apologies. It seems the one liner that fixes this wasn't checked in on the reactive streams branch. This should be fixed in the next incremental release.

Regards.

**Comment by alexflav23** [comment]
Hi @mmatloka,

This is now fixed, it will be available in 1.16.0 to be released later today.

Regards.

**Comment by mmatloka** [comment]
Thx!
